.circleci/config.yml

```diff
@@ -1,43 +1,48 @@
 version: 2
 
+references:
+  # Ancilliary containers
+  postgres-container: &postgres-container
+    image: postgres:9.6
+    environment:
+      POSTGRES_USER: routemaster
+      POSTGRES_DB: routemaster
+      POSTGRES_PASSWORD: routemaster
+
+
 jobs:
-  test:
+  test-36: &test-template
```

Comment:
Could be worth extracting the test-template out to the top level, or in `references`, so that all 3 tests look the same?

Reply:
yeah, especially now I've got a `references` anyway, that's a good call. 662c535